src/Aperture/overlap.jl

```diff
@@ -154,3 +154,117 @@ function area_arc(x0, y0, x1, y1, r)
     θ = 2asin(0.5a / r)
     return 0.5r^2 * (θ - sin(θ))
 end
+
+"""
+Utility function to find whether a point is inside ellipse or not.
```

Comment:
The docstring should start with the function call definition, like what you did for example for `EllipticalAperture`